scripts/Units/cormexp.bos

```diff
@@ -5,99 +5,51 @@
 
 piece base, column, top, turret1, turret2, barrel, flare1, flare2, flare3, aimpoint;
 
-static-var  ExtractionSpeed, ArmorUp, restore_delay,
-            gun_1, gun_2, Static_Var_8, bAiming, statechg_DesiredState, statechg_StateChanging;
+static-var  ExtractionSpeed, restore_delay,
+            gun_1, gun_2, statechg_DesiredState, statechg_StateChanging, column_going_up, weapon_state;
```

Comment:
`statechg_DesiredState` and `statechg_StateChanging` are unused and can be removed

Reply:
addressed